.github/workflows/js.yml

```
@@ -30,17 +28,34 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: yarn
     - run: yarn downloadCustomTypes
-    - run: yarn lint
     - run: yarn test
-
+    - run: yarn build
+  integration:
+    name: Integration test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+    - name: Cancel Previous Runs
```

```suggestion
    - name: Cancel Previous Runs

```
please add the spaces before the new job everywhere, it's easier to read